### DISCUSSION TITLE
`JULIA_ENABLE_THREADING` is a ghost-documented option?

### DISCUSSION THREAD

**Discussion by 13steinj** [JuliaLang/julia#51959] [General]
`JULIA_ENABLE_THREADING` doesn't do anything since a18ab97f74dce6f9d116690bff89ddd73c0ce308 between v1.2.0 and v1.3.0; but in https://github.com/JuliaLang/julia/commit/65b8e7ed72541d1dd5bc1f4dcabc5245e1dba718#diff-f3b2f102b60ee9cb4051b53a77df35f99735d25c4a41553d4eaa913e46177ea0R10 (part of v1.5.0 onwards) references it. Is this supposed to reference something else? be removed entirely?

I'd have made an issue but the warning text there said that issues are only for "bug reports" and some people don't consider this kind of thing a bug report. Feel free to convert to an issue.

**Comment by vtjnash** [comment]
Bad comments are also bugs. A PR would also be welcome to fix it.
